frontend/src/components/utils/Button.js

```diff
@@ -50,9 +77,17 @@ export const Button = styled(ButtonComponent)`
   background-color: #f0f0f0;
   border-color: #f0f0f0;
 
+  ::before {
+    ${({ tooltip }) => tooltip && `content: "${tooltip}"; ${ButtonTooltip}`}
+  }
+
   :hover {
     background-color: #e2e6ea;
     border-color: #e2e6ea;
+
```

Comment:
need this space?

Reply:
Yes, because there is more code comming after the space